apps/example-orchestrator-ui/components/Subscriptions/subscriptionsQuery.ts

```diff
@@ -0,0 +1,83 @@
+import { TypedDocumentNode } from '@graphql-typed-document-node/core';
+import {
+    PythiaSortOrder,
+    SubscriptionGridQuery,
+    SubscriptionGridQueryVariables,
+    SubscriptionsSort,
+} from '../../__generated__/graphql';
+import { graphql } from '../../__generated__';
+
+export const GET_SUBSCRIPTIONS_PAGINATED: TypedDocumentNode<
+    SubscriptionGridQuery,
+    SubscriptionGridQueryVariables
+> = graphql(`
+    query SubscriptionGrid(
+        $first: Int!
+        $after: Int!
+        $sortBy: [SubscriptionsSort!]
+        $filterBy: [[String!]!]
+    ) {
+        subscriptions(
+            first: $first
+            after: $after
+            sortBy: $sortBy
+            filterBy: $filterBy
+        ) {
+            edges {
+                node {
+                    note
+                    name
+                    startDate
+                    endDate
+                    tag
+                    vlanRange
+                    description
+                    product {
+                        name
+                        type
+                        tag
+                    }
+                    organisation {
+                        abbreviation
+                        name
+                    }
+                    insync
+                    status
+                    subscriptionId
+                }
+            }
+        }
+    }
+`);
+
+export const DEFAULT_SORT_FIELD = 'startDate';
+export const DEFAULT_SORT_ORDER = PythiaSortOrder.Desc;
+
+export const DEFAULT_SUBSCRIPTIONS_SORT_ORDER: SubscriptionsSort = {
+    field: DEFAULT_SORT_FIELD,
+    order: DEFAULT_SORT_ORDER,
+};
+
+export const GET_SUBSCRIPTIONS_PAGINATED_DEFAULT_VARIABLES: SubscriptionGridQueryVariables =
+    {
+        first: 20,
+        after: 0,
+        sortBy: DEFAULT_SUBSCRIPTIONS_SORT_ORDER,
+    };
+
+export const getPythiaSortOrderFromString = (
```

Comment:
Pythia will soon be no more, also it's not a publicly available app; rename to "Graphql" (a todo is fine of course)

Reply:
True, when Pythia is no more, the return type of this function will change too. Once that happens I will update the name of this function too.